_posts/2025-02-06-highlights-2024.md

```diff
@@ -26,6 +26,8 @@ A few of the highlights covered in this issue:
 
 ### Named tuples and other Scala 3 improvements
 
+*Powered by LAMP, the Scala open-source team at VirtusLab, the Scala team at Akka and the Scala Center*
```

Comment:
@tgodzik  Is it okay if I shorten `the Scala open-source team at VirtusLab` to `the Scala team at VirtusLab`?